portal.go

```diff
@@ -879,9 +893,18 @@ func (portal *Portal) HandleMatrixReaction(evt *event.Event) {
 		existing.MXID = evt.ID
 		existing.Update()
 	}
+
+	if errorMsg != "" {
+		portal.bridge.AS.SendErrorMessageSendCheckpoint(evt, appservice.StepRemote, errors.New(errorMsg), true, 0)
+	}
 }
 
 func (portal *Portal) HandleMatrixRedaction(evt *event.Event) {
+	if !portal.bridge.IM.Capabilities().SendTapbacks {
+		portal.sendUnsupportedCheckpoint(evt, appservice.StepRemote, errors.New("Reaction is not supported in portal"))
```

Comment:
```suggestion
		portal.sendUnsupportedCheckpoint(evt, appservice.StepRemote, errors.New("Bridge does not support any kinds of redactions"))
```

Reply:
This isn't true, though... it supports them in certain conversations.